### DISCUSSION TITLE
Mobile forensics

### DISCUSSION THREAD

**Discussion by Scriptkiddie8** [Textualize/textual#50] [Q&A]
.

**Comment by willmcgugan** [comment]
Haven't the foggiest. Have you posted in the right repo?

**Reply by Scriptkiddie8** [reply]
Sorry?

**Reply by willmcgugan** [reply]
Forgiven.

**Reply by Scriptkiddie8** [reply]
Sir, I'm new here so just need a bit of time to know the correct way to use GitHub, sorry if I've done something wrong

**Reply by willmcgugan** [reply]
Not at all. But I don't know anything about mobile forensics or UFED. I think you have come to the wrong place if you need help with those.

**Reply by Scriptkiddie8** [reply]
So where I should go now, who can help me out?

**Reply by willmcgugan** [reply]
Try the search function top left? Or google.
